content/doc/book/pipeline/syntax.adoc

```diff
@@ -889,6 +889,60 @@ pipeline {
 // Script //
 ----
 
+=== Parallel
+
+Stages in Declarative Pipeline may declare a number of nested stages within
+them, which will be executed in parallel. Note that a stage must have one and
+only one of either `steps` or `parallel`. The nested stages may use all other
+configuration available for stages in Declarative Pipeline, including `when`,
+`environment`, `agent`, `post`, and `tools`, but are required to have `steps`
```

Comment:
This could probably be simplified to state that parallel stages cannot contain nested parallel statements but otherwise behave just as the stage directive does

Reply:
Okiedokie.